apps/build-onchain-apps/app/home/index.tsx

```diff
@@ -15,6 +16,7 @@ export default function HomePage() {
         <WhyUseIt />
         <Guide />
       </main>
+      <Footer />
```

Comment:
Can we make this part of the Layout? And have the footer component everywhere.

Reply:
I'm having a weird createContext function not defined when I try to render the footer component within layout.tsx. I believe this is caused by Radix. cc @wespickett 

![image](https://github.com/coinbase/build-onchain-apps/assets/43140758/6862c8f2-cca9-45de-b402-8acc60da034b)

I'm just going to add the component to all pages for now...

Reply:
After syncing with @wespickett, we made it work within layout.tsx by adding `'use client'` at the top of the footer component code. Check 3a93743.